src_c/_sdl2/touch.c

```diff
@@ -140,13 +140,13 @@ MODINIT_DEFINE(touch)
                                          NULL};
     import_pygame_base();
     if (PyErr_Occurred()) {
-        MODINIT_ERROR;
+        return NULL;
     }
 
     /* create the module */
     module = PyModule_Create(&_module);
     if (module == NULL) {
-        MODINIT_ERROR;
```

Comment:
does this need a module decref?

Reply:
no, because `module` is already `NULL` here